components/Button/button.css

```diff
@@ -66,55 +66,67 @@
 
 
 .btn-xs{
-    font-size: 1.25rem;
-    height: 2.5rem;
+    font-size: 1.7rem;
+    height: 3.5rem;
     min-width: 3.5rem;
     max-width: max-content;
-    padding-inline-start: 0.75rem;
-    padding-inline-end: 0.75rem;
+    padding-inline-start: 1.25rem;
+    padding-inline-end: 1.25rem;
```

Comment:
any specific reason to use `padding-inline-start`, `padding-inline-end` and not `padding-left` , `padding-right`?

Reply:
Its just a short-hand property. No specific reason

Reply:
I think there is a slight difference between padding and padding-inline - https://css-tricks.com/almanac/properties/p/padding-inline/#aa-values . But if you are aware of it, all good.

Reply:
Yes I know the difference. Thankyou for pointing out.